src/main/java/uk/gov/hmcts/dts/fact/model/Court.java

```diff
@@ -88,8 +89,15 @@ public Court(uk.gov.hmcts.dts.fact.entity.Court courtEntity) {
                 return areaOfLawObj;
             })
             .collect(toList());
-        this.contacts = courtEntity.getContacts().stream().filter(NAME_IS_NOT_DX)
-            .map(Contact::new).collect(toList());
+        final List<uk.gov.hmcts.dts.fact.entity.Contact> contacts = ofNullable(courtEntity.getCourtContacts())
```

Comment:
Instead of returning a null, would an empty list be better in the case that no contacts are found on the database? 

I am just thinking that if we are looping through a list on the frontend anyways, it saves time and effort when it comes to checking if variables and responses are null or not. 

In which case, a logger message can also be added in the backend and potentially the frontend to indicate that no contacts were found as well.

Reply:
Ah it returns an empty list, my bad!